tests/spread/cross-compile/stage-packages/snap/snapcraft.yaml

```diff
@@ -18,6 +18,7 @@ parts:
       - xxd
     - to armhf:
       - tar:armhf
+    - on amd64 to armhf:
```

Comment:
what is being tested or asserted here?

Reply:
We want to make sure that "on <host arch> to <target arch>" works. The bug we fixed used target arch for both.